### DISCUSSION TITLE
enbrighten GE In-Wall Smart Switch not identified

### DISCUSSION THREAD

**Discussion by Userblues** [zwave-js/zwave-js#7258] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
After inclusion I get an unknown manufacturer 0xXXXX, product 0xXXXX, device 0xXXXX

Anything else we should know?
I do not see this switch in products.z-wavealliance.org, any hope to get it working, ever?

### Software versions

zwave-js-ui: 9.19.0.572035a
zwave-js: 13.3.1
home id: 25478258
home hex: 0x184c472

### Z-Wave Controller (Stick/Dongle/...)

AEON Labs	Z-Stick S2 USB Controller	DSA02203

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_current.log.gz](https://github.com/user-attachments/files/17299901/zwavejs_current.log.gz)

**Comment by AlCalzone** [comment]
The device isn't responding at all to the queries from Z-Wave JS. Check out https://zwave-js.github.io/node-zwave-js/#/troubleshooting/first-steps and follow it. You can skip the section **Reduce reports from the network** for now, as it doesn't look like this is a problem in your mesh.

**Comment by Userblues** [comment]
I've reached conclusion this device is DOA. Dagnabbit, it was a spare in my drawer for 3+ years, now the warranty is expired.
